core/UI/itembar.py

```diff
@@ -61,8 +61,17 @@ def draw(self):
         pygame.draw.rect(self.game.screen, (50, 50, 50), itembarrects[6])
         self.game.screen.blit(pygame.transform.scale(Assets.BACKPACK1.value.image, (60, 60)), itembar[7])
 
-        self.helditem = (Settings.Game.WIDTH / 2 - 44, Settings.Game.HEIGHT / 2 - 6)
         for i, item in enumerate(self.game.player.inventory.getslots()):
             self.game.screen.blit(pygame.transform.scale(item.item.image, (60, 60)), itembar[i])
+            if item.item != Items.EMPTY.value:
+                if 1 <= item.quantity <= 9:
+                    amount = pygame.font.SysFont('Corbel', 25).render(str(item.quantity), True, Colors.WHITE)
```

Comment:
Ik zou persoonlijk de amount = ... buiten de ifs halen aangezien die 3x t zelfde is, maar maakt niet heel veel uit

Reply:
ja true, had ik niet gezien maar is gedaan